src/utilities/FormManager.ts

```diff
@@ -82,7 +104,7 @@ class FormManager {
   }
 
   public get formFieldNames () {
-    return Object.keys(flatten<{ [key: string]: any }, { [key: string]: any }>(this.formValues));
+    return Object.keys(flattenObject<{ [key: string]: any }, { [key: string]: any }>(this.formValues));
```

Comment:
Is this fixing a known bug or something ? Or it's just that ... it cause no issues but it should have been like this from the beginning ?

Reply:
The problem is that there are 2 libraries which both have 'flatten' as their suggested function name, and this file uses both. I could have just renamed one of them, but renaming both seemed the clearest option:
![image](https://user-images.githubusercontent.com/796717/53113595-f6a06880-350f-11e9-9fb4-8b57fcab67aa.png)

Reply:
So to clarify, not a bug-fix, not the way it always should have been (eh, maybe that one), but a differentiation because now we have 2 flatten functions in this file.